lib/tasks/race_condition.rake

```diff
@@ -0,0 +1,33 @@
+task 'race_condition' => :environment do |t|
```

Comment:
Did you mean to check this in? It seems like a test case that might not need to be in core?

Reply:
oops, good catch.